.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -31,7 +31,7 @@ assignees: ''
  - [ ] [DIS] Upload release APK and bundle to internal owncloud instance
  - [ ] [GIT] Merge master branch into stable, in owncloud/android-library
  - [ ] [GIT] Merge master branch into stable, in owncloud/android
- - [ ] [DOC] Update owncloud.org/download version numbers (notify rocketchat #marketing)
```

Comment:
You may want to keep the notification for marketing because of other marketing dependent stuff
(dont know if it is relevant or not).

The notification at the doc-internal channel is definitely to be added 👍